src/ios/CDVInAppBrowser.m

```diff
@@ -1149,6 +1192,20 @@ - (id)init
 + (CDVInAppBrowserOptions*)parseOptions:(NSString*)options
 {
 	CDVInAppBrowserOptions* obj = [[CDVInAppBrowserOptions alloc] init];
+	NSNumberFormatter* numberFormatter = [[NSNumberFormatter alloc] init];
+	[numberFormatter setAllowsFloats:YES];
+
+	//Set the Default value of the objects
+	[obj setValue:@"top" forKey:@"toolbarposition"];
+	[obj setValue:@"Close" forKey:@"closebuttoncaption"];
+	[obj setValue:[NSNumber numberWithBool:NO] forKey:@"disallowoverscroll"];
+	[obj setValue:[NSNumber numberWithBool:NO] forKey:@"shownavigationbtns"];
+	[obj setValue:[numberFormatter numberFromString:@"0xFFFFFF"] forKey:@"closebuttoncolor"];
+	[obj setValue:[numberFormatter numberFromString:@"0xFF0000"] forKey:@"gradient1"];
+	[obj setValue:[numberFormatter numberFromString:@"0xFF0000"] forKey:@"gradient2"];
+	[obj setValue:[numberFormatter numberFromString:@"0xFF0000"] forKey:@"toolbarbgcolor"];
+	[obj setValue:[numberFormatter numberFromString:@"1.0"] forKey:@"alphagradient1"];
```

Comment:
The original implementation uses the numberFormatter to obtain the number from the string. I thought by using the same implementation, it will reduce confusion and add to code flow.